micropython/net/webrepl/webrepl.py

```diff
@@ -7,19 +7,71 @@
 import socket
 import sys
 import websocket
-import _webrepl
+import io
 
 listen_s = None
 client_s = None
 
 DEBUG = 0
 
-_DEFAULT_STATIC_HOST = const("https://micropython.org/webrepl/")
+_DEFAULT_STATIC_HOST = const("https://felix.dogcraft.de/webrepl/")
```

Comment:
To maintain the old webrepl webpage for older devices, does it make sense to change this URL to something like `https://micropython.org/webrepl2/`?  Or is it enough to just specify different JS code as you've done below with the line `webreplv2_content.js`?  Ie, can both the old and new web clients exist at this same URL?

Reply:
For people who open the webrepl web-client like I do, by navigating to the micropython-device and then being served [this webpage](https://github.com/micropython/micropython-lib/blob/master/micropython/net/webrepl/webrepl.py#L87), the micropython device can select the correct file and from there the correct dependencies. These clients can choose the correct files and exist at the same URL.

It does make a difference for people who navigate to `https://micropython.org/webrepl/` manually and then enter their device's IP address in the text field. `https://micropython.org/webrepl/` can only serve one version. How many people do that? I am not directly aware of documentation that sends people directly to that URL. Do you have some access statistics of how often `https://micropython.org/webrepl/`  is accessed vs `https://micropython.org/webrepl/webrepl_content.js`? Also those people who manually navigate to the website would need to be aware that there are two different versions and then choose the correct one.

I would argue that we recommend people to navigate to their device directly so it can choose the correct version and don't recommend to navigate to the page on `micropython.org` manually.

The main "pro"-argument for keeping the same base url would be, that we can "eventually" switch the version served there to show webreplv2, so we start catching those people who navigate to the URL (from wherever they might have found the link).

So that's the rationale for why I chose this variant, but I have no strong preference at all for either.

Reply:
OK, let's keep it as the same URL.